File: src/main/java/brs/db/store/Dbs.java
package brs.db.store;

import brs.db.BlockDb;
import brs.db.PeerDb;
import brs.db.TransactionDb;

public interface Dbs { // TODO remove, integrate with dependencyProvider
  BlockDb getBlockDb();

  TransactionDb getTransactionDb();

  PeerDb getPeerDb();
}

File: src/main/java/brs/db/store/Dbs.kt
package brs.db.store

import brs.db.BlockDb
import brs.db.PeerDb
import brs.db.TransactionDb

interface Dbs { // TODO remove, integrate with dependencyProvider
    val blockDb: BlockDb

    val transactionDb: TransactionDb

    val peerDb: PeerDb
}
